.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build & Test
+on:
+  push:
+    branches:
+      - master
+      - release-*
+  pull_request:
+env:
+  DOTNET_NOLOGO: true
+jobs:
+  build:
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ windows-2019, ubuntu-18.04 ]
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.x
+      - name: Setup .NET Core 3.1 runtime
```

Comment:
This is a bit misleading, since it doesn't just set up the runtime, it sets up the SDK. Although, it _should_ be masked by the previous 5.0 installation. For safety, I would perform the 3.1 set up first, and then the 5.0 set up, to ensure the later installer gets precedence (and I would rename this step to say "SDK" rather than "runtime").

Reply:
The names are intended to be what we _need_ from it, not a full description of everything it _does_. The ordering doesn't matter, but we prefer the SDK first.